core-services/prow/02_config/_plugins.yaml

```diff
@@ -1577,6 +1593,28 @@ plugins:
   - approve
   heketi/heketi:
   - trigger
+  integr8ly/ansible-tower-configuration:
+  - assign
+  - blockade
+  - cat
+  - dog
+  - heart
+  - golint
+  - help
+  - hold
+  - label
+  - lgtm
+  - lifecycle
+  - override
```

Comment:
Can you add the retitle plugin here, i just opened this to add it to the rest https://github.com/openshift/release/pull/6880